modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/render.tsx

```diff
@@ -47,9 +47,15 @@ export default function render(
 	},
 	container: Element
 ) {
+
+	// @ts-ignore
```

Comment:
Because `Liferay` does not exist on the `window` object. I remembered [this Slack thread](https://liferay.slack.com/archives/C3JBR21HA/p1637919004194300) and went with ignoring

Reply:
This was probably complaining about `Liferay` not existing on the `window`  object. Some later changes made it irrelevant I guess 😕